app/core/logging.py

```
@@ -0,0 +1,255 @@
+                    "method": method,
+                    "status": status_code or 0,
+                    "latency_ms": latency_ms,
+                    "model_id": settings.DEFAULT_MODEL_ID or "",
+                    "backend": settings.MODEL_BACKEND,
+                }
+                # Record metrics (best-effort; do not raise)
+                try:
+                    inc_http_request(path, method, str(status_code or 0))
+                    observe_request_latency(path, method, str(status_code or 0), latency_ms)
+                except Exception:
+                    pass
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)